client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -29,7 +39,7 @@ function Dashboard() {
           </Link>
         )
       }
-      contentClassName={styles['panels']}
+      contentClassName={`${styles['panels']} count--${panelCount}`}
```

Comment:
Brilliant. Could also apply something like this to determine whether items are in the first or second column, apply a custom class, and apply that "small width" class to those elements, rather than specify sysmon panel and custom panel

Reply:
We are on the exact same page.